src/platform/cmx/user/cos_types.h

```diff
@@ -0,0 +1,920 @@
+/**
+ * Copyright 2014 by Gabriel Parmer, gparmer@gwu.edu
+ *
+ * Redistribution of this file is permitted under the GNU General
+ * Public License v2.
+ */
+
+/*
+ * Note that this file is included both by the kernel and by
+ * components.  Unfortunately, that means that ifdefs might need to be
+ * used to maintain the correct defines.
+ */
+
+#ifndef TYPES_H
+#define TYPES_H
+
+#include "chal/consts.h"
+#include "chal/cos_config.h"
+
```

Comment:
What is changed in this file?

The updated file (in the head on github) removes a lot of crap.